Multisig/UI/SwiftUI/Settings/Get In Touch/EmailSupportViewController.swift

```diff
@@ -32,8 +32,8 @@ struct EmailSupportViewController: UIViewControllerRepresentable {
         let chain = safe != nil ? "\(safe!.chain!.name!) (\(safe!.chain!.id!))" : "None"
 
         let message = """
-        Safe \(version)
-        Safe address: \(address)
+        Safe Account \(version)
```

Comment:
I think, this is about mobile app version, not Safe Account version. Could you check?